atom/browser/browser_linux.cc

```diff
@@ -50,6 +51,10 @@ bool Browser::IsDefaultProtocolClient(const std::string& protocol,
   return false;
 }
 
+base::string16 Browser::GetDefaultProtocolClient(const std::string& protocol) {
+  return base::UTF8ToUTF16("");
```

Comment:
Calling `UTF8ToUTF16` is inefficient, you should just `return base::string16();`